.eslintrc.js

```diff
@@ -23,13 +23,9 @@ module.exports = {
         {
             files: ['*.test.js', '*.test.tsx'],
             globals: {
-                blurInput: true,
-                clearInput: true,
+                BoxVisualTestUtils: true,
```

Comment:
Might create an importable helper module for tests later, so that this global isn't needed